### PR TITLE
Edited Logic Blocks in the APIS

### PR DESCRIPTION
**Pull request by shivabalachandran**
Changes the Logic blocks in the API files.

### CODE REVIEW

File: modules/extensions/eprocess/publisher/eprocess/apis/processes.jag [hunk 1]
```diff
@@ -10,22 +10,25 @@ require('/modules/publisher.js').exec(function(ctx) {
...
+		if(user){
 			var am=asset.createUserAssetManager(ctx.session,type);
 			var query={};
 			query.overview_name=q;
 			var assets=am.search(query,null);
 			print(assets);
+
+		}
+		else{
+			log.error('User is not authenticated/ authorized to perform the request');
```
Comment (imesh) [inline modules/extensions/eprocess/publisher/eprocess/apis/processes.jag]: May be the space between "/" and "authorized" can be removed.

File: modules/extensions/eprocess/store/eprocess/apis/processes.jag [hunk 2]
```diff
@@ -27,22 +27,25 @@ require('/modules/store.js').exec(function(ctx) {
...
-		else{
+		if(user){
 			var am = asset.createUserAssetManager(ctx.session,type);
 			var query = {};
 			query.overview_name = q;
 			var assets = am.search(query,null);
 			print(assets);
 		}
+		else{
+			log.error('User is not authenticated/ authorized to perform the request');
```
Comment (imesh) [inline modules/extensions/eprocess/store/eprocess/apis/processes.jag]: The same issue mentioned above.

**Comment by shivabalachandran** [comment]
Fixed in PR https://github.com/wso2/product-pc/pull/11. Thank you.

File: modules/extensions/eprocess/publisher/eprocess/apis/processes.jag (hunk 1 above)
Comment (shivabalachandran) [inline modules/extensions/eprocess/publisher/eprocess/apis/processes.jag]: Fixed in PR https://github.com/wso2/product-pc/pull/11. Thank you.

File: modules/extensions/eprocess/store/eprocess/apis/processes.jag (hunk 2 above)
Comment (shivabalachandran) [inline modules/extensions/eprocess/store/eprocess/apis/processes.jag]: Fixed in PR #11. Thank you.

**Pull request merged**